draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -564,7 +600,12 @@ in (#hash_function_claim)).
 
 ### Validating the Key Binding JWT
 
-To validate the signature on the Key Binding JWT, the Verifier MUST use the key material in the SD-JWT. If it is not clear from the SD-JWT, the Key Binding JWT MUST specify which key material the Verifier needs to use to validate the Key Binding JWT signature using JOSE header parameters such as `kid` and `x5c`.
+The Verifier MUST ensure that the key with which it validates the signature on
+the Key Binding JWT is the key specified in the SD-JWT as the using the Holder's
```

Comment:
seem to  be too many words in this sentence - does this still say what you wanted to say? 
```suggestion
The Verifier MUST ensure that the key with which it validates the signature on
the Key Binding JWT is the key specified in the SD-JWT as the Holder's
```